packages/sample-app/src/components/common/PageLayout.tsx

```diff
@@ -1,9 +1,9 @@
 import { Page, PageSection } from '@patternfly/react-core';
 import * as React from 'react';
 
-type PageLayoutProps = {
+type PageLayoutProps = React.PropsWithChildren<{
```

Comment:
FYI, the reason I've used `PropsWithChildren` here is to better communicate the fact that this component is supposed to have children.

In React 17, `React.FC` type automatically uses `PropsWithChildren` which adds `children?: ReactNode | undefined` even if the component is _not_ supposed to have children by design (so React is basically imposing a hard assumption).

This is one of the downsides of using `React.FC` as mentioned in https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript - for example, `<Foo>something</Foo>` expression is valid even if the `Foo` component is not supposed to have children :smiley: 

This issue should be fixed in React 18.
- https://solverfox.dev/writing/no-implicit-children/
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210 (see "Removal of implicit children")